q2_diversity/markdown/core_metrics.md

```diff
@@ -0,0 +1,94 @@
+outputs:
+    - faith_pd_vector:
+        - AlphaDiversity
+        - pandas.Series
+    - observed_otus_vector:
+        - AlphaDiversity
+        - pandas.Series
+    - unweighted_unifrac_distance_matrix:
+        - DistanceMatrix
+        - skbio.DistanceMatrix
+    - weighted_unifrac_distance_matrix:
+        - DistanceMatrix
+        - skbio.DistanceMatrix
+    - jaccard_distance_matrix:
+        - DistanceMatrix
+        - skbio.DistanceMatrix
+    - bray_curtis_distance_matrix:
+        - DistanceMatrix
+        - skbio.DistanceMatrix
+    - unweighted_unifrac_pcoa_results:
+        - PCoAResults
+        - skbio.OrdinationResults
+    - weighted_unifrac_pcoa_results:
+        - PCoAResults
+        - skbio.OrdinationResults
+    - jaccard_pcoa_results:
+        - PCoAResults
+        - skbio.OrdinationResults
+    - bray_curtis_pcoa_results:
+        - PCoAResults
+        - skbio.OrdinationResults
+---
+## Core diversity metrics
+
+This method rarefies a feature table to an even sampling depth and then applies a suite of alpha and beta diversity metrics. The metrics applied are:
+ * Alpha diversity
+  * Faith's Phylogenetic Diversity index: a qualitative phylogenetic measure of community richness
+  * Observed OTUs: a qualitative non-phylogenetic measure of community richness
+ * Beta diversity
+  * Unweighted UniFrac: a qualitative phylogenetic measure of pairwise community dissimilarity (computed for all pairs of samples)
+  * Weighted UniFrac: a quantitative phylogenetic measure of pairwise community dissimilarity (computed for all pairs of samples)
+  * Jaccard: a qualitative non-phylogenetic measure of pairwise community dissimilarity (computed for all pairs of samples)
+  * Bray-Curtis: a quantitative non-phylogenetic measure of pairwise community dissimilarity (computed for all pairs of samples)
+
+Principle coordinates analysis is subsequently applied to all pairwise distance matrices resulting from the beta diversity computations.
+
+### Rarefy feature table
+
+```python
+>>> from q2_feature_table import rarefy
+>>> rarefied_table = rarefy(feature_table, depth=depth)
+```
+
+### Compute alpha diversity metrics
+
+```python
+>>> from q2_diversity import alpha, alpha_phylogenetic
+>>> faith_pd_vector = alpha_phylogenetic('faith_pd', rarefied_table, phylogeny=phylogeny)
+>>> observed_otus_vector = alpha('observed_otus', rarefied_table)
+```
+
+### Compute beta diversity metrics
+
+```python
+>>> from q2_diversity import beta, beta_phylogenetic
+>>> unweighted_unifrac_distance_matrix = beta_phylogenetic('unweighted_unifrac', rarefied_table, phylogeny=phylogeny)
+>>> weighted_unifrac_distance_matrix = beta_phylogenetic('weighted_unifrac', rarefied_table, phylogeny=phylogeny)
+>>> jaccard_distance_matrix = beta('jaccard', rarefied_table)
+>>> bray_curtis_distance_matrix = beta('braycurtis', rarefied_table)
+```
+
+### Apply principal coordinate analysis to beta diversity results
+
+```python
+>>> import skbio.stats.ordination
```

Comment:
It'd be cool to use `q2_diversity.pcoa` here in case that function diverges from skbio (and it'd match what you're doing in the previous steps)